packages/common/src/utils/extractCookies.ts

```diff
@@ -14,15 +14,11 @@
  * limitations under the License.
  */
 
-import {
-  calculateEffectiveExpiryDate,
-  type CompleteJson,
-  type CookieFrameStorageType,
-} from '@google-psat/common';
-
 /**
  * Internal dependencies
  */
+import { CompleteJson, CookieFrameStorageType } from '../cookies.types';
```

Comment:
Please use `type`

Reply:
Fixed in [d2aca92](https://github.com/GoogleChromeLabs/ps-analysis-tool/pull/814/commits/d2aca929c6b25a68d828a6bd79e4e422e6b91e5f)